code/menuui/readyroom.cpp

```diff
@@ -999,6 +1008,7 @@ void sim_room_init()
 	int i;
 	sim_room_buttons *b;
 	char wild_card[256];
+	bool dummy_buffer;
```

Comment:
Can you move this to the `#ifdef` block where this is needed? Then it will not appear in release builds.

Reply:
Good point.